### ISSUE TITLE
about DISFA F1score

### ISSUE THREAD

**Issue by vhzy** [CVI-SZU/ME-GraphAU#9]
Hello! I followed the image example you provided for pre-processing and got the same image database. 
However after completing the training for stage2 using resnet50, the three folds average is less than 60.
I  used your model for test and found it to be consistent with what the paper reported, even higher actually.
Here are some details:
Using the second fold (provided in your code) for training, the highest score was probably about 56 after completing stage 1 and about 58 after completing stage 2.
I wonder if i missed something in the training process?Thank you!

**Comment by yangchengjun** [comment]
I also have very low accuracy, even with the author's model, and the command and F1 scores are as follows:
```sh
python test.py --dataset DISFA --arc resnet50 --exp-name MEflod1_ResNet50_in_DISFA --fold 2  --resume results/ME-GraphAU_resnet50_DISFA/MEFARG_resnet50_DISFA_fold1.pth
```
2024-01-30 23:50:26,927:INFO: {'val_mean_f1_score 40.76 val_mean_acc 83.07'}
2024-01-30 23:50:26,927:INFO: {'F1-score-list:'}
2024-01-30 23:50:26,927:INFO: {'AU1: 20.54 AU2: 2.18 AU4: 64.23  AU6: 69.65 AU9: 45.61 AU12: 39.35  AU25: 52.82 AU26: 31.67 '}
2024-01-30 23:50:26,927:INFO: {'Acc-list:'}
2024-01-30 23:50:26,927:INFO: {'AU1: 77.06 AU2: 96.91 AU4: 84.93  AU6: 95.05 AU9: 96.36 AU12: 89.27  AU25: 46.12 AU26: 78.88 '}
```sh
python test.py --dataset DISFA --arc resnet50 --exp-name Test_MEflod2_ResNet50_in_DISFA --fold 2  --resume results/ME-GraphAU_resnet50_DISFA/MEFARG_resnet50_DISFA_fold2.pth
```

{'val_mean_f1_score 19.68 val_mean_acc 90.37'}
{'F1-score-list:'}
{'AU1: 13.39 AU2: 0.00 AU4: 11.35  AU6: 22.51 AU9: 0.00 AU12: 38.10  AU25: 70.18 AU26: 1.92 '}
{'Acc-list:'}
{'AU1: 93.32 AU2: 97.98 AU4: 84.49  AU6: 91.23 AU9: 97.05 AU12: 83.29  AU25: 84.94 AU26: 90.63 '}
```sh
python test.py --dataset DISFA --arc resnet50 --exp-name Test_MEflod3_ResNet50_in_DISFA --fold 2  --resume results/ME-GraphAU_resnet50_DISFA/MEFARG_resnet50_DISFA_fold3.pth
```
{'val_mean_f1_score 39.95 val_mean_acc 86.33'}
{'F1-score-list:'}
{'AU1: 21.50 AU2: 26.07 AU4: 43.63  AU6: 53.44 AU9: 39.78 AU12: 43.95  AU25: 69.83 AU26: 21.38 '}
{'Acc-list:'}
{'AU1: 77.82 AU2: 96.38 AU4: 81.47  AU6: 92.63 AU9: 97.46 AU12: 86.98  AU25: 76.13 AU26: 81.78 '}

**Comment by MUHAMMAD-FARMAN** [comment]
@vhzy @yangchengjun The DISFA dataset contains videos from two camera angles. Did you use both or just one? If you used both, what was the folder structure?

**Comment by Dharmendra1995** [comment]
Hi, I used just one right side camera angle. I obtain the following for DISFA fold 2:

{'F1-score-list:'}
{'AU1: 16.40 AU2: 19.90 AU4: 25.81  AU6: 25.43 AU9: 8.79 AU12: 36.88  AU25: 42.59 AU26: 10.75 '}
{'Acc-list:'}
{'AU1: 90.67 AU2: 92.28 AU4: 75.35  AU6: 82.08 AU9: 92.63 AU12: 77.84  AU25: 60.84 AU26: 79.79 '}

This is just for epoch 3, but worried why it is getting such a low F1 Score. 

100%|████████████████████████████████████████████████████████████████████████████| 682/682 [01:21<00:00,  8.35it/s]
{'Epoch:  1   train_loss: 0.06046  val_loss: 0.39896  val_mean_f1_score 22.80,val_mean_acc 81.36'}
{'F1-score-list:'}
{'AU1: 9.14 AU2: 18.24 AU4: 27.02  AU6: 25.12 AU9: 10.66 AU12: 37.59  AU25: 41.55 AU26: 13.09 '}
{'Acc-list:'}
{'AU1: 91.47 AU2: 92.31 AU4: 74.92  AU6: 78.22 AU9: 92.62 AU12: 70.78  AU25: 63.17 AU26: 87.39 '}
Epoch: [2 | 20 LR: 9.938531812030452e-05 ]
100%|██████████████████████████████████████████████████████████████████████████| 1363/1363 [06:53<00:00,  3.30it/s]
100%|████████████████████████████████████████████████████████████████████████████| 682/682 [01:21<00:00,  8.36it/s]
{'Epoch:  2   train_loss: 0.02790  val_loss: 0.42754  val_mean_f1_score 23.32,val_mean_acc 81.43'}
{'F1-score-list:'}
{'AU1: 16.40 AU2: 19.90 AU4: 25.81  AU6: 25.43 AU9: 8.79 AU12: 36.88  AU25: 42.59 AU26: 10.75 '}
{'Acc-list:'}
{'AU1: 90.67 AU2: 92.28 AU4: 75.35  AU6: 82.08 AU9: 92.63 AU12: 77.84  AU25: 60.84 AU26: 79.79 '}
Epoch: [3 | 20 LR: 9.755460614005131e-05 ]
100%|██████████████████████████████████████████████████████████████████████████| 1363/1363 [06:52<00:00,  3.30it/s]
100%|████████████████████████████████████████████████████████████████████████████| 682/682 [01:21<00:00,  8.35it/s]
{'Epoch:  3   train_loss: 0.02085  val_loss: 0.42482  val_mean_f1_score 21.08,val_mean_acc 82.33'}
{'F1-score-list:'}
{'AU1: 8.98 AU2: 13.53 AU4: 25.21  AU6: 18.65 AU9: 8.80 AU12: 38.39  AU25: 43.66 AU26: 11.39 '}
{'Acc-list:'}
{'AU1: 91.49 AU2: 92.09 AU4: 73.74  AU6: 86.02 AU9: 92.45 AU12: 73.98  AU25: 62.16 AU26: 86.69 '}
